bigbluebutton-html5/imports/api/breakouts-history/server/handlers/breakoutRoomsList.js

```diff
@@ -0,0 +1,35 @@
+import { check } from 'meteor/check';
+import BreakoutsHistory from '/imports/api/breakouts-history';
+import Logger from '../../../../startup/server/logger';
+
+export default function handleBreakoutRoomsList({ body }) {
+  const {
+    meetingId,
+    rooms,
+  } = body;
+
+  check(meetingId, String);
+
+  const selector = {
+    meetingId,
+  };
+
+  const modifier = {
+    $set: {
+      meetingId,
```

Comment:
If we selected based on the meetingId, should we overwrite it?

Reply:
The idea was to keep only last breakouts data.
It overwrites every time new breakout rooms are created.